package.json

```diff
@@ -41,7 +41,7 @@
   "dependencies": {
     "@gnosis.pm/dex-contracts": "^0.3.2",
     "@gnosis.pm/owl-token": "^3.1.0",
-    "@gnosis.pm/safe-contracts": "github:gnosis/safe-contracts#v1.1.1-dev.1",
+    "@gnosis.pm/safe-contracts": "github:gnosis/safe-contracts#v1.1.1-dev.2",
```

Comment:
Was this published just for us!?

Reply:
The changes that we need in `safe-contract` will officially be part of future versions, so we will be able to update to newer versions without any issue in the future!